test/async.coffee

```diff
@@ -204,3 +204,21 @@ test "#3199: await multiline implicit object", ->
         type: 'a'
         msg: 'b'
     eq undefined, y
+
+test "top-level await", ->
+  eq CoffeeScript.compile('await null'), '''
+    (async function() {
+      await null;
+
+    }).call(this);
+
+  '''
```

Comment:
We have the `eqJS` helper that’s more concise, and does `compile` with `bare`:

```suggestion
  eqJS ‘await null’, ‘await null;’
```

Reply:
Ah, thanks, I figured you must have a helper like that.

I think there's value in testing both the bare and non-bare case, given that I had to write code to make the wrapper `async`.  Do you agree?  If so, should I include both tests like this, or make/modify helper to support nonbare?

Reply:
Let’s make separate tests for the non-bare wrapper, I think that’s worth calling out. I had no idea that’s what you were testing here.

I don’t think we need to modify the helper just for this one case. You can just use `CoffeeScript.compile` to test the wrapper.